src/test/java/com/exasol/errorcodecrawlermavenplugin/config/ErrorCodeConfigReaderTest.java

```diff
@@ -23,6 +23,15 @@ void testRead() throws IOException, ErrorCodeConfigException {
         copyResourceToTestProject("errorCodeConfig/valid.yml");
         final ErrorCodeConfig read = new ErrorCodeConfigReader(this.tempDir).read();
         assertThat(read.getPackagesForErrorTag("EXM"), containsInAnyOrder("com.exasol.example"));
+        assertThat(read.getHighestIndexForErrorTag("EXM"), equalTo(6));
+    }
+
+    @Test
+    void testReadFileWithMissingHighestIndexReturnsZero() throws IOException, ErrorCodeConfigException {
+        copyResourceToTestProject("errorCodeConfig/validMissingHighestIndex.yml");
```

Comment:
not sure if it's worth moving this to a resource since it's only a few lines.

Reply:
Inlined config files into test ✅